### DISCUSSION TITLE
Sending Photo in TeleDart

### DISCUSSION THREAD

**Discussion by shakibm83** [DinoLeung/TeleDart#182] [Q&A] [answered]
how can I send photo with url?

this is my code:
![Untitled](https://user-images.githubusercontent.com/77733647/135524503-fe3fe551-8d14-4bb4-af60-e6ce9c51d3a6.png)

I give a error.

```
Unhandled exception:
TelegramException: Attribute 'photo' can only be either io.File or String (Telegram file_id or image url)
```
I am completely sure that photo's url is right

**Comment by DinoLeung** [comment, marked as answer]
Are you completely sure the image url string includes the protocol as well as host domain name?

**Reply by shakibm83** [reply]
Thank you for answering🙏
I had a mistake. photo variable was a class and I must use **ToString()** function.
**BTW** the way for sending a image is like this:
`await event.replyPhoto(url) // url like this https://cdn.pixabay.com/photo/2015/04/23/22/00/tree-736885_960_720.jpg`
